sdk/core/core-http/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Release History
 
```

Comment:
```suggestion
- Use `self` instead of `window` when referring to the global object in a browser context, since `window` might not always be defined. [PR 13873](https://github.com/Azure/azure-sdk-for-js/pull/13873)
```